src/scipp/html/formatting_datagroup_html.py

```diff
@@ -11,25 +13,68 @@
 from ..core.cpp_classes import DataArray, Dataset, Variable
 from ..core.data_group import DataGroup
 from ..units import dimensionless
-from .resources import load_atomic_tpl, load_collapsible_tpl, load_dg_detail_tpl, \
-    load_dg_repr_tpl, load_dg_style
+from .resources import load_atomic_row_tpl, load_collapsible_row_tpl, \
+    load_dg_detail_list_tpl, load_dg_repr_tpl, load_dg_style
 
 
-def _make_shape_repr(var: Union[Variable, DataArray, Dataset, DataGroup]) -> str:
+class mathstr(str):
```

Comment:
This part is an example that I wanted to share.

I thought it would be nice to allow \$ equation \$ string...? Like this...?

Reply:
I don't think we can do that in general. There may be strings that contain $ signs that are not meant to be equations.

Reply:
For now I updated it to escape all `$` to keep it as a string, but when it's loaded by the `Jupyter`, it still wraps `$` within `<span>` html tag. 
I don't know why, but I just had to update css to remove padding in the span element because of this.